code/framework/api/src/main/java/io/cattle/platform/api/resource/jooq/AbstractJooqResourceManager.java

```diff
@@ -377,7 +376,7 @@ public boolean handleException(Throwable t, ApiRequest apiRequest) {
         if (t instanceof ProcessCancelException) {
             log.info("Process cancel", t.getMessage());
             throw new ClientVisibleException(ResponseCodes.CONFLICT);
-        } else if (t instanceof DataAccessException || t instanceof DataChangedException) {
+        } else if (t instanceof DataAccessException) {
```

Comment:
DataChangedException already extends DataAccessException, so that fix of mine was fixing nothing. Reverted it.